app/src/main/java/com/github/se/travelpouch/ui/fields/locationQueryField.kt

```diff
@@ -0,0 +1,86 @@
+package com.github.se.travelpouch.ui.fields
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.heightIn
+import androidx.compose.foundation.layout.padding
+import androidx.compose.material3.DropdownMenu
+import androidx.compose.material3.DropdownMenuItem
+import androidx.compose.material3.HorizontalDivider
+import androidx.compose.material3.OutlinedTextField
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.MutableState
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import androidx.compose.ui.window.PopupProperties
+import com.github.se.travelpouch.model.location.LocationViewModel
+import com.github.se.travelpouch.model.travels.Location
+
+@Composable
+fun locationInputField(
```

Comment:
It would be better to start with an uppercase letter to follow the naming conventions :  `LocationInputField`